tests/ui_tools/test_popups.py

```diff
@@ -590,6 +594,7 @@ def test_keypress_navigation(self, mocker, widget_size,
             'subscribers': [],
             'date_created': 1472047124,
             'stream_weekly_traffic': 140,
+            'email_address': 'zt@chat.zulip.org',
```

Comment:
In practice these email addresses can be very long. A separate test with that kind of example would be useful. For the reason of long emails, I would suggest we consider allowing/forcing the URL onto the next line. For example, try applying your PR to the #announce stream on czo.

Reply:
I understand your concern. I'll fix this too.